common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -1517,7 +1514,7 @@ private static String normalizeOs(String value) {
         if (value.startsWith("linux")) {
             return "linux";
         }
-        if (value.startsWith("macosx") || value.startsWith("osx")) {
+        if (value.startsWith("macosx") || value.startsWith("osx") || value.startsWith("darwin")) {
```

Comment:
`Native` class for KQueue uses `darwin` as well: https://github.com/netty/netty/pull/11239/files#diff-0f0a0e330d992ea697cf2a6def937bc08caadca2007948010f4eb22c2c84cf75L139